src/Symfony/Component/Filesystem/LockHandler.php

```diff
@@ -11,6 +11,8 @@
 
 namespace Symfony\Component\Filesystem;
 
+@trigger_error(sprintf('The %s class is deprecated since version 3.4 and will be removed in 4.0. Use the SemaphoreStore or FlockStore class instead.', LockHandler::class), E_USER_DEPRECATED);
```

Comment:
getting this notice while using the standalone filesystem, I would try to find the SemaphoreStore in the filesystem component. I would use FQCNs or prepend `Lock` here